src/style.css

```diff
@@ -1,3 +1,59 @@
+/* stylelint-disable custom-property-empty-line-before */
+/* stylelint-disable comment-empty-line-before */
+/* stylelint-disable font-family-name-quotes */
 body {
   margin: 0;
+  font-family: 'Roboto', serif;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  position: relative;
+  justify-content: flex-end;
+  background-color: #fff;
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+  line-height: 60px;
+  text-transform: uppercase;
+  font-size: 12;
```

Comment:
The `font-size` property is missing a unit. Consider adding a unit like `px`, `em`, or `rem` to specify the font size correctly.